ui/src/app/views/profile-edit/component.css

```diff
@@ -1,3 +1,25 @@
 h2 {
   display: inline-block;
 }
+
+.field-wrapper {
+  margin: 0.5rem 0 1.5rem 0;
+}
+
+label {
+  display: block;
```

Comment:
Something to keep in mind, usability-wise. Block display fills the width of the component's parent and clicking a label focuses the attached input, checks the attached box, etc. Block display can sometimes cause the clickable area to be too large causing unintentional clicks.